src/CXml/Model/Description.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace CXml\Model;
+
+use JMS\Serializer\Annotation as Ser;
+
+class Description extends MultilanguageString
+{
+	/**
+	 * @Ser\SerializedName("ShortName")
+	 * @Ser\XmlElement(cdata=false)
```

Comment:
Are u sure that ShorName should not be rendered as CData?

Reply:
Ok, we can avoid CDATA in this case. ;)

In CI we have a step `Run Tests`: why it say ok? Normally the xml generated should not more match with sample xml
`tests/metadata/cxml/samples/PunchoutOrderMessage.xml` or the method `assertXmlStringEqualsXmlFile` does not take care on CDATA?